cmd/kubebuilder/initproject/vendor.go

```diff
@@ -142,4 +144,45 @@ func RunVendorInstall(cmd *cobra.Command, args []string) {
 			log.Fatalf("Could not write file %s: %v", p, err)
 		}
 	}
+
+	err = updateDepConfig()
+	if err != nil {
+		log.Fatalf("Could not update Gopkg.toml file: %v", err)
+	}
+}
+
+// updateDepConfig updates the Dep config Gopkg.toml to include Kubebuilder
+// project dependency. It uses the Kubebuilder version to determine whether
+// to include branch or version in the contraint stanza.
+func updateDepConfig() error {
```

Comment:
What usage does the generated project make of kubebuilder as a vendored dep? Does it depend on kubebuilder/pkg?

I have been wondering about this, still as a relative newcomer without a lot of hands on experience.